packages/concerto-core/lib/decoratormanager.js

```diff
@@ -253,6 +253,252 @@ class DecoratorManager {
         return newModelManager;
     }
 
+    /**
+     * Extracts all the decorator commands from all the models in modelManager
+     * @param {ModelManager} modelManager the input model manager
+     * @param {object} options - decorator models options
+     * @param {boolean} options.removeDecoratorsFromModel - flag to strip out decorators from models
+     * @param {string} options.locale - locale for extracted vocabulary set
+     * @returns {Object} a new model manager with the decorations removed and a list of extracted decorator jsons
+     */
+    static extractDecorators(modelManager,options) {
+        options = {
+            removeDecoratorsFromModel: false,
+            locale:'en',
+            ...options
+        };
+        const ast = modelManager.getAst(true);
+        const decoratedAst = JSON.parse(JSON.stringify(ast));
+        let extractionDictionary={};
```

Comment:
```suggestion
        let extractionDictionary = {};
```

Reply:
hey @muskanbararia can you please run a linter to fix the formatting niggles? Thanks!

Reply:
@Jonathan-Casey this reports no errors when I run `npm run lint`. do you want me to add new rules to eslintrc?